superset/views/base.py

```diff
@@ -141,6 +141,9 @@ def api(f):
     def wraps(self, *args, **kwargs):
         try:
             return f(self, *args, **kwargs)
+        except SupersetSecurityException as ex:
```

Comment:
Shouldn't this be managed by also wrapping the function in `handle_api_exception`  as opposed to adding it here?